web/controllers/user_controller.ex

```diff
@@ -34,6 +35,21 @@ defmodule CodeCorps.UserController do
     end
   end
 
+  def github_connect(conn, code) do
+    current_user = Guardian.Plug.current_resource(conn)
+    with {:ok, user} <- Github.connect(current_user, code)
+    do
+      # {:ok, user}
```

Comment:
@begedin I tried doing what you suggested with just sending back the `{:ok, user}` and having `ja_resource` handle it, but was running into trouble with `ja_resource` recognizing the custom action. If you have a suggestion on how to remedy this that would be great!